scripts/i.in.spotvgt/i.in.spotvgt.py

```diff
@@ -155,7 +155,8 @@ def main():
         s = p.communicate()[0]
         if s == "application/x-zip":
             gscript.fatal(_("Please extract %s before import.") % infile)
-    except:
+    # code changed here, Exception included instead of bare except
```

Comment:
No need for the text. This should be in the commit message. Extend PR description if needed. That's what we can use when writing the commit message when merging the PR.